lc0/src/chess/board.cc

```diff
@@ -169,6 +169,8 @@ static const Move::Promotion kPromotions[] = {
     Move::Promotion::Knight,
 };
 
+static const BitBoard kPawnMask = 0x00FFFFFFFFFFFF00ULL;
```

Comment:
Static is not necessary inside anonymous namespace. Not a large difference, but we don't have it elsewhere. :)

Reply:
You have it in all the declarations above it :) So I thought I'd be consistent.

Reply:
Oops sorry, too much nitpicking from my side. :)